diff/parse.go

```diff
@@ -304,7 +304,7 @@ func (r *FileDiffReader) ReadExtendedHeaders() ([]string, error) {
 			var err error
 			line, err = readLine(r.reader)
 			if err == io.EOF {
-				return xheaders, &ParseError{r.line, r.offset, ErrExtendedHeadersEOF}
+				return xheaders, ParseError{r.line, r.offset, ErrExtendedHeadersEOF}
```

Comment:
isn't this line exactly the one we're testing for in diff_test.go? Why does Codecov say that this line is not covered by tests?